scripts/start.js

```diff
@@ -100,15 +102,22 @@ choosePort(HOST, DEFAULT_PORT)
         }
       ];
     var ganache = require("ganache-cli");
-    console.log(`Starting ganache on port ${process.env.DEV_GANACHE_PORT}`)
-    var ganacheServer = ganache.server({port:process.env.DEV_GANACHE_PORT,network_id:0,accounts});
-    ganacheServer.listen(process.env.DEV_GANACHE_PORT, function(err, blockchain) {
-    
-      if (err){
+    console.log(`Starting ganache on port ${process.env.DEV_GANACHE_PORT}`);
+    var ganacheServer = ganache.server({ port: process.env.DEV_GANACHE_PORT, network_id: 0, accounts });
+    ganacheServer.on('error', function (err) {
+      if (err.code && err.code==='EADDRINUSE') {
+        console.log(`Port ${process.env.DEV_GANACHE_PORT} in use. Assuming a ganache instance on that port.`);
+      } else {
+        throw err;
+      }
+    });
+    ganacheServer.listen(process.env.DEV_GANACHE_PORT, process.env.DEV_GANACHE_HOST, function (err, blockchain) {
```

Comment:
@lalexgap if there is already another server instance running on the port, what happens here? Is this line needed?

Reply:
Nope, it should only be called if the port is not in use